Makefile.js

```diff
@@ -279,17 +279,16 @@ target.gensite = function() {
 
     echo("Generating eslint.org");
 
-    if (test("-d", DOCS_DIR)) {
-        rm("-r", DOCS_DIR);
```

Comment:
Removing the old directory was the only way to ensure files deleted from the two were also deleted from the site. How will this be done now?